docs/fleet/provision.md

```diff
@@ -135,6 +135,16 @@ The Viam Agent is installed as a `systemd` service named `viam-agent`.
 
   For more information, see [Viam Agent management](https://github.com/viamrobotics/agent#management).
 
+### View Viam Agent logs
+
+The Viam Agent writes log messages to the [Viam app](https://app.viam.com/).
+These messages can be found on the [**LOGS** tab](/cloud/machines/#logs) of your machine's page.
+
+Log messages are only sent when your machine is online and connected to the internet.
```

Comment:
```suggestion
Viam Agent only sends messages when your machine is online and connected to the internet.
```
[opt] passive voice
I'm not actually sure of accuracy of my suggestion, but would be great to say what is sending the messages